src/main/webapp/app/exam/participate/exercises/file-upload/file-upload-exam-submission.component.ts

```diff
@@ -166,4 +167,7 @@ export class FileUploadExamSubmissionComponent extends ExamSubmissionComponent i
     private onError() {
         this.alertService.error(this.translateService.instant('error.fileUploadSavingError'));
     }
+    setSubmissionVersion(submissionVersion: SubmissionVersion): void {
```

Comment:
Maybe add a `// @ts-ignore` here?
Additionally you could throw an error or log something in the console when this method is invoked, since it should not be used, should it?

Reply:
no, it shouldn't because submission versions simply don't exist for file upload exercises and I don't expect this to change in follow-ups as we would need to store a lot of files on the file system if we introduce submission versions for them.

Reply:
add the ts-ignore because it's unused or what's the rationale behind that?

Reply:
Yes, if it's intentionally unused, we would get rid of the error here